zebra-chain/Cargo.toml

```diff
@@ -97,10 +97,13 @@ orchard = "0.8.0"
 zcash_encoding = "0.2.0"
 zcash_history = "0.4.0"
 zcash_note_encryption = "0.4.0"
-zcash_primitives = { version = "0.15.0", features = ["transparent-inputs"] }
+# TODO: Revert to a release once librustzcash is released (#8749).
+zcash_primitives = { git = "https://github.com/zcash/librustzcash/", branch = "main", features = ["transparent-inputs"] }
 sapling = { package = "sapling-crypto", version = "0.1" }
-zcash_protocol = { version = "0.1.1" }
-zcash_address = { version = "0.3.2" }
+# TODO: Revert to a release once librustzcash is released (#8749).
+zcash_protocol = { git = "https://github.com/zcash/librustzcash/", branch = "main"}
+# TODO: Revert to a release once librustzcash is released (#8749).
+zcash_address = { git = "https://github.com/zcash/librustzcash/", branch = "main"}
```

Comment:
I think it's preferrable to point to a commit in the main branch, and not the main branch itself, e.g. 

`{ git = "https://github.com/zcash/librustzcash/", rev = "5a4a3e06dcd2cf5bdf79a7cd48709b58693c65f0" }`

This will make it less likely to suddenly break if something incompatible is merged into their `main`.